apps/settings/src/components/AppList/AppScore.vue

```diff
@@ -21,22 +22,69 @@
   -->
 
 <template>
-	<img :src="scoreImage" :alt="t('settings', 'Rating: {score}/10', {score:appScore})" class="app-score-image">
+	<span role="img"
+		:aria-label="title"
+		:title="title"
+		class="app-score__wrapper">
+		<NcIconSvgWrapper v-for="index in fullStars"
+			:key="`full-star-${index}`"
+			:path="mdiStar"
+			inline />
+		<NcIconSvgWrapper v-if="hasHalfStar" :path="mdiStarHalfFull" inline />
+		<NcIconSvgWrapper v-for="index in emptyStars"
+			:key="`empty-star-${index}`"
+			:path="mdiStarOutline"
+			inline />
+	</span>
 </template>
-<script>
-import { imagePath } from '@nextcloud/router'
+<script lang="ts">
+import NcIconSvgWrapper from '@nextcloud/vue/dist/Components/NcIconSvgWrapper.js'
+import { mdiStar, mdiStarHalfFull, mdiStarOutline } from '@mdi/js'
+import { translate as t } from '@nextcloud/l10n'
+import { defineComponent } from 'vue'
 
-export default {
+export default defineComponent({
 	name: 'AppScore',
-	props: ['score'],
-	computed: {
-		appScore() {
-			return Math.round(this.score * 10)
+	components: {
+		NcIconSvgWrapper,
+	},
+	props: {
+		score: {
+			type: Number,
+			required: true,
 		},
-		scoreImage() {
-			const imageName = 'rating/s' + this.appScore + '.svg'
-			return imagePath('core', imageName)
+	},
+	setup() {
+		return {
+			mdiStar,
+			mdiStarHalfFull,
+			mdiStarOutline,
 		}
 	},
-}
+	computed: {
+		title() {
+			const appScore = (this.score * 5).toFixed(1)
+			return t('settings', 'Community rating: {score}/5', { score: appScore })
+		},
+		fullStars() {
+			return Math.floor(this.score * 5 + 0.25)
+		},
+		emptyStars() {
+			return Math.min(Math.floor((1 - this.score) * 5 + 0.25), 5 - this.fullStars)
+		},
+		hasHalfStar() {
+			return (this.fullStars + this.emptyStars) < 5
+		},
+	},
+})
 </script>
+<style scoped>
+.app-score__wrapper {
+	display: inline-flex;
+	color: var(--color-favorite, #a08b00);
```

Comment:
```suggestion
	color: var(--color-favorite, #a08b00);
```
Will be added as a new variable in followup?